doc_analyzer.go

```diff
@@ -40,6 +43,10 @@ func docAnalyze(path string, logLevel int) {
 	renderer := html.NewRenderer(opts)
 	md, err := ioutil.ReadFile(path)
 	if err != nil {
+		if strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
use [os.IsNotExist](https://golang.org/pkg/os/#IsNotExist) instead to skip the nil-err check
```go
if os.IsNotExist(err) {
 // ...
}
```